lib/inspec_tools/inspec.rb

```diff
@@ -191,6 +140,32 @@ def generate_ckl
       @checklist.asset = generate_asset
     end
 
+    def generate_vuln_data(control)
+      vuln = HappyMapperTools::StigChecklist::Vuln.new
+      stig_data_list = []
+
+      stig_data_list = handle_vuln_num(stig_data_list, control)
```

Comment:
The pattern of passing the `stig_data_list` in every time seems a little unnecessary here. We could just use `<<` and then call `.reject!(&:nil?)` on line 159. None of the methods use the `stig_data_list` so it is just an unnecessary parameter every single time.